open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/handlers/SubjectAreaGlossaryHandler.java

```diff
@@ -340,4 +330,40 @@ private void updateAttributes(Glossary oldGlossary, Glossary newGlossary) {
         }
         return response;
     }
+
+    /**
+     * Get Glossary terms
+     *
+     * @param userId     unique identifier for requesting user, under which the request is performed
+     * @param guid       guid of the category to get terms
+     * @return A list of terms for the glossary
+     * when not successful the following Exception responses can occur
+     * <ul>
+     * <li> UserNotAuthorizedException           the requesting user is not authorized to issue this request.</li>
+     * <li> InvalidParameterException            one of the parameters is null or invalid.</li>
+     * <li> PropertyServerException              Property server exception. </li>
+     * </ul>
+     * */
+    public SubjectAreaOMASAPIResponse<Term> getGlossaryTerms(String userId, String guid) {
```

Comment:
This method (and all the other get relationship APs you have added) should have paging options. I notice that the existing get relationships do not have the paging options - I would prefer new APIs to be added with paging , you could fix the existing get relationship APIs to add paging here as well if you like.